src/components/Resources/ResourceCard.tsx

```diff
@@ -1,24 +1,32 @@
 import { ChainModifiers, Entry } from 'contentful'
-import {
-  ResourceCategoryType,
-  ResourceLinkSkeleton,
-} from 'src/types/contentful-types'
+import { ResourceLinkSkeleton } from 'src/types/contentful-types'
 import CategoryPill from './CategoryPill'
 import { Card } from 'react-bootstrap'
+import { ResourceTagId, resourceTagLabelMap } from './resourceTagLabelMap'
+import { useRollbar } from '@rollbar/react'
 
 export default function ResourceCard({
   resource,
 }: {
   resource: Entry<ResourceLinkSkeleton, ChainModifiers, string>
   index: number
 }) {
+  const rollbar = useRollbar()
   const url = resource.fields.url as string
   const title = resource.fields.title as string
   const organization = resource.fields.organization as string
   const description = resource.fields.description as string
-  const category = (
-    resource.fields.category as string[]
-  )[0] as ResourceCategoryType
+
+  const tagLabels: string[] = resource.metadata.tags.map((t) => {
+    const label = resourceTagLabelMap[t.sys.id as ResourceTagId]
+    if (!label) {
```

Comment:
We want this to notify us so we can update the resourceTagLabelMap if a new tag has been added that we haven't accounted for.